docs/UserGuide.md

```diff
@@ -206,117 +174,120 @@ Date:   Fri Feb 9 19:13:13 2018 +0800
-- Clicking on the ramp will redirect user to the GitHub page, which contains all the commits within the fixed time period.
+`repo-config.csv` file contains repo-level config data as follows:
 
-#### Contribution Bar
-The total amount of code contributed is represented by the **red bars**, and the length of these red bars is proportional to the total contribution of the corresponding author.
-Hovering over the bar shows the exact amount of contribution.
-If the author contributes **too much** compared to other authors, there will be multiple red bars in his `Chart Panel`.
+* First row: column headings, ignored by RepoSense
+* Second row: repo data
+
+Here is an example:
+
+Repository's Location|Branch|File formats|Ignore Glob List|Ignore standalone config|Ignore Commits List
+---------------------|------|------------|----------------|------------------------|-------------------
+`https://github.com/foo/bar.git`|`master`|`java;css`|`test/**`|`yes`|`2fb6b9b2dd9fa40bf0f9815da2cb0ae8731436c7;c5a6dc774e22099cd9ddeb0faff1e75f9cf4f151`
+
+<br>
+
+Column Name | Explanation
+----------- | -----------
+Repository's Location | The `GitHub URL` or `Disk Path` to the git repository e.g., `https://github.com/foo/bar.git` or `C:\Users\user\Desktop\GitHub\foo\bar`
+Branch | The branch to analyze in the target repository e.g., `master`. Default: the default branch of the repo
+[Optional] File formats<sup>*</sup> | The file extensions to analyze. Default: `adoc;cs;css;fxml;gradle;html;java;js;json;jsp;md;py;tag;xml`
+[Optional] Ignore Glob List<sup>*</sup> | The list of file path globs to ignore during analysis for each author. e.g., `test/**;temp/**`
+[Optional] Ignore standalone config | To ignore the standalone config file (if any) in target repository, enter **`yes`**. If the cell is empty, the standalone config file in the repo will take precedence over configurations provided in the csv files.
+[Optional] Ignore Commit List<sup>*</sup> | The list of commits to ignore during analysis. For accurate results, the commits should be provided with their full hash.
+
+<sup>* **Multi-value column**: multiple values can be entered in this column using a semicolon `;` as the separator.</sup>
+
+#### `author-config.csv`
+
+Optionally, you can use a `author-config.csv` (which should be in the same directory as `repo-config.csv` file) to provide more details about the authors to analyze ([example](author-config.csv)). It should contain the following columns:
+
+Column Name | Explanation
+----------- | -----------
+Repository's Location | Same as `repo-config.csv`
+Branch | The branch to analyze for this author
+Author's GitHub ID | GitHub username of the target author e.g., `JohnDoe`
+[Optional] Author's Display Name | The name to display for the author. Default: author's GitHub username.
+[Optional] Author's Git Author Name<sup>*</sup> | Detailed explanation below
```

Comment:
The explanation of `Git Author Name` is above now, perhaps provide a link here too.

Reply:
Done